src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity.php

```diff
@@ -502,6 +495,25 @@ public static function fromManageResponse(
         return $entity;
     }
 
+    private static function setRedirectUrisFromManageResponse(
+        Entity $entity,
+        ManageEntity $manageEntity,
+        $environment,
+        $playGroundUriTest = '',
```

Comment:
If you add input validation in `\Surfnet\ServiceProviderDashboard\Application\Service\LoadEntityService::__construct` using `Assert::stringNotEmpty` you can kind of prevent the `$playGroundUriTest = ''` in this method. Although it's not guaranteed that the LoadEntityService only uses this static constructor.

Reply:
When I first created the optional parameters in the entity domain entity it was not possible to make them mandatory. Because the oidc logic is now completed I'm able to make them mandatory. Thanks for pointing that out.